### PR TITLE
Add file resources containing video resources, blogposts and Slides

### LINKED ISSUE #329: list out video resource about DVC in docs
I recently found out various informative videos about DVC in youtube. I felt it would ideal to mention about these video resources in DVC documentation as well.

- https://www.youtube.com/watch?v=4h6I9_xeYA4
- https://www.youtube.com/watch?v=jkfh2PM5Sz8

### PR DESCRIPTION
**Pull request by kurianbenoy**
Fix #329 

@shcheklein do suggest changes and lets add more resources into it. I had created this page with having [django restframework resources in mind](https://www.django-rest-framework.org/community/tutorials-and-resources/). Now I feel we can add good blog posts and even slides also in this section, what do you think @shcheklein @efiop ?

### CODE REVIEW

**Comment by kurianbenoy** [comment]
Incorporated the changes, I will add blog posts and tutorials you mentioned sometime later today.

File: static/docs/philosophy/video-resources.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## Video Resources
```
Comment (shcheklein) [inline static/docs/philosophy/video-resources.md]: It should be single # not ##

File: static/docs/philosophy/video-resources.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+## Video Resources 
+
+### Talks by DVC team
```
Comment (shcheklein) [inline static/docs/philosophy/video-resources.md]: subsections should be ## so that we show them in the right nav bar

Comment (shcheklein) [reply static/docs/philosophy/video-resources.md]: if it's the only subsection of the Video Resources, let's merge it withe the title

File: src/Documentation/sidebar.json [hunk 3]
```diff
@@ -186,15 +186,16 @@
     }
   },
   {
-    "name": "DVC Philosophy",
+    "name": "Understanding DVC",
```
Comment (shcheklein) [inline src/Documentation/sidebar.json]: I would rename the directory itself as well. The one that contains all these files.

Comment (kurianbenoy) [reply src/Documentation/sidebar.json]: changed the folder name as well :)

**Comment by kurianbenoy** [comment]
Screenshot-2019-5-17 Data Science Version Control System
On running the server up in local environment, the following error came

File: src/Documentation/sidebar.json [hunk 4]
```diff
@@ -41,8 +41,7 @@
     "labels": {
       "data-and-model-files-versioning.md": "Data & Model Files Versioning",
       "share-data-and-model-files.md": "Share Data & Model Files",
-      "multiple-data-scientists-on-a-single-machine.md":
-        "Shared Development Machine"
+      "multiple-data-scientists-on-a-single-machine.md": "Shared Development Machine"
```
Comment (shcheklein) [inline src/Documentation/sidebar.json]: I believe this line now is > 80 symbols. Please, don't forget to run prettier for JS and MD files.

**Comment by shcheklein** [comment]
Thanks @kurianbenoy !! 🎉

**Pull request merged**